### PR TITLE
Updated README with recent changes

### PR DESCRIPTION
**Pull request by sthirugn**

### CODE REVIEW

**Comment by omaciel** [comment]
ACK

File: docs/index.rst [hunk 1]
```diff
@@ -134,7 +130,10 @@ Some basic usage::
 
     testimony validate tests/
 
-For further information about the options check Testimony's help.
+Misc:
```
Comment (elyezer) [inline docs/index.rst]: Add a blank line below, I think the way it is Sphinx will not generate a list properly. Also you can remove indention of the beginning of each list item.

**Comment by elyezer** [comment]
ACK, pending comments.

File: docs/index.rst (hunk 1 above)
Comment (sthirugn) [inline docs/index.rst]: thanks, updated.

**Comment by sthirugn** [comment]
rebased after the other pr

File: docs/index.rst [hunk 2]
```diff
@@ -134,7 +130,11 @@ Some basic usage::
...
+
+1. a json output is provided when ``--json`` option is specified.
+2. a colored output is provided when ``termcolor`` package is installed. This
+     can be disabled by specifying ``--no-color`` option.
```
Comment (elyezer) [inline docs/index.rst]: need to fix the indentation

**Pull request merged**